example_test.go

```diff
@@ -151,3 +152,63 @@ func ExampleDrivesBuilder_DriveOpt() {
 		panic(err)
 	}
 }
+
+func ExampleNetworkInterfaces_RateLimiting() {
+	// construct the limitations of the bandwidth for firecracker
+	bandwidthB := firecracker.TokenBucketBuilder{}.
+		WithInitialSize(1024 * 1024).
```

Comment:
I'm not super wild about this naming.  The interface is very generic, but it makes it hard to see what the limiter is limiting; is this constrained to 1024*1024 bytes of bandwidth per 30s?  I have to rely on the name of the variable `bandwidthB` to guess what it's limiting and infer that the limit unit is bytes.

Reply:
Yea, that makes sense. I've went ahead and made the changes.